pages/index.tsx

```diff
@@ -115,29 +112,46 @@ const ConnectWithMe = () => {
     <div className="flex items-center h-fit bg-gray-200/40 backdrop-blur-sm border-2 border-zinc-100 w-fit  px-2 py-2  rounded-tl-full rounded-bl-full pr-10 translate-x-40 hover:translate-x-6  transition">
       {/* linkedin connect button with icon  */}
       <div className="flex flex-row m-auto items-center space-x-4">
-      <Image className="rounded-full" alt="LinkedIn" width="60" height="60" src="https://avatars.githubusercontent.com/u/79275157?v=4"/>
-        <div onClick={() => 
-    // go to linked in page
-    window.location.href="https://www.linkedin.com/in/piyush-kalyan/"
-    } className="cursor-pointer" >
-        <span className="flex flex-row w-fit gap-2 bg-[#2885e0] p-2 text-white font-medium rounded-full">
-          <div>
-          <LinkedInIcon/>
-          </div>
-          <a  href="https://www.linkedin.com/in/piyush-kalyan/">Linked In</a>
-        </span>
-   
+        <Image
+          className="rounded-full"
+          alt="LinkedIn"
+          width="60"
+          height="60"
+          src="https://avatars.githubusercontent.com/u/79275157?v=4"
+        />
+        <div
+          onClick={() =>
+            // go to linked in page
+            (window.location.href =
+              "https://www.linkedin.com/in/piyush-kalyan/")
+          }
+          className="cursor-pointer"
+        >
+          <span className="flex flex-row w-fit gap-2 bg-[#2885e0] p-2 text-white font-medium rounded-full">
+            <div>
+              <LinkedInIcon />
+            </div>
+            <a href="https://www.linkedin.com/in/piyush-kalyan/">Linked In</a>
+          </span>
         </div>
-
       </div>
     </div>
-  )
-}
+  );
+};
 
 const LinkedInIcon = () => {
   return (
-    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" data-supported-dps="24x24" fill="currentColor" className="mercado-match " width="24" height="24" focusable="false">
-  <path d="M20.5 2h-17A1.5 1.5 0 002 3.5v17A1.5 1.5 0 003.5 22h17a1.5 1.5 0 001.5-1.5v-17A1.5 1.5 0 0020.5 2zM8 19H5v-9h3zM6.5 8.25A1.75 1.75 0 118.3 6.5a1.78 1.78 0 01-1.8 1.75zM19 19h-3v-4.74c0-1.42-.6-1.93-1.38-1.93A1.74 1.74 0 0013 14.19a.66.66 0 000 .14V19h-3v-9h2.9v1.3a3.11 3.11 0 012.7-1.4c1.55 0 3.36.86 3.36 3.66z"></path>
-</svg>
-  )
-}
\ No newline at end of file
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 24 24"
+      data-supported-dps="24x24"
+      fill="currentColor"
+      className="mercado-match "
+      width="24"
+      height="24"
+      focusable="false"
+    >
+      <path d="M20.5 2h-17A1.5 1.5 0 002 3.5v17A1.5 1.5 0 003.5 22h17a1.5 1.5 0 001.5-1.5v-17A1.5 1.5 0 0020.5 2zM8 19H5v-9h3zM6.5 8.25A1.75 1.75 0 118.3 6.5a1.78 1.78 0 01-1.8 1.75zM19 19h-3v-4.74c0-1.42-.6-1.93-1.38-1.93A1.74 1.74 0 0013 14.19a.66.66 0 000 .14V19h-3v-9h2.9v1.3a3.11 3.11 0 012.7-1.4c1.55 0 3.36.86 3.36 3.66z"></path>
+    </svg>
+  );
+};
```

Comment:
Here also, there are no code changes.

Reply:
I have done no code changes here, there was just auto indentation (prettier extension) in vs code which indented the code in neat lines according to it's algo.